modules/backend/lang/th/lang.php

```diff
@@ -0,0 +1,628 @@
+<?php
+
+return [
+    'auth' => [
+        'title' => 'ส่วนการจัดการ',
+        'invalid_login' => 'รายละเอียดที่คุณเพิ่มเข้ามาไม่ตรงกับข้อมูลของเรา กรุณาตรวจสอบและลองอีกครั้ง',
+    ],
+    'field' => [
+        'invalid_type' => 'ประเภทของฟิลด์ไม่ถูกต้อง :type.',
+        'options_method_invalid_model' => "The attribute ':field' does not resolve to a valid model. Try specifying the options method for model class :model explicitly.",
+        'options_method_not_exists' => "The model class :model must define a method :method() returning options for the ':field' form field.",
+        'colors_method_not_exists' => "The model class :model must define a method :method() returning html color HEX codes for the ':field' form field.",
```

Comment:
Untranslated strings should be left out of the language files so that future updates to the default language (English) will propagate properly @anurat

Reply:
I see. Is there anyway to keep the untranslated fields so that I am able to update them later on?

Reply:
You could in theory comment them out with inline comments but I'm not really a fan of that. I would like it if there was an official plugin to review the language files and solicit missing translations from the community.